packages/api-integration/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@withorbit/api-integration",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "private": true,
+  "type": "module",
+  "sideEffects": false,
+  "devDependencies": {
+    "@types/jest": "^26.0.24",
+    "@types/node": "^16.4.3",
+    "babel-jest": "^27.0.6",
+    "jest": "^27.0.6",
+    "typescript": "^4.3.5"
+  },
+  "dependencies": {
+    "isomorphic-fetch": "^3.0.0"
+  },
+  "scripts": {
+    "test": "HOST=http://localhost:5001/metabook-system/us-central1 jest"
```

Comment:
If you think it'd be cleaner, the `backend` tests have some code which actually starts up and tears down the Firebase emulator within the test execution: https://github.com/andymatuschak/orbit/blob/master/packages/backend/src/backend/firebaseSupport/__tests__/firebaseTesting.ts. Though if you wanted to use that you might have to move this into the `backend` package.